pykeeb/keyswitch_mount.py

```diff
@@ -7,7 +7,7 @@ class Keyswitch_mount:
 	alps_keyswitch = Import('/usr/models/matias.stl').color('Gray')
 	mx_keyswitch = Import('/usr/models/cherry.stl').color('Gray')
 	dsa_key = Import('/usr/models/dsa_1u.stl').color('White') #18x18x8mm 
-	def __init__(self, transformations, ik=False, switch_type='alps', mount_length=DSA_KEY_WIDTH, mount_width=DSA_KEY_WIDTH):
+	def __init__(self, transformations, ik=False, switch_type='mx', mount_length=DSA_KEY_WIDTH, mount_width=DSA_KEY_WIDTH):
```

Comment:
Haha, if this project gets some serious attention I'll change it.

Reply:
I was not able to find a way to change the switch type in the keyboard_matrix class.  This was my solution. Should probably implement a more flexible method.